source/checksums.c

```diff
@@ -30,22 +34,16 @@ napi_value crc_common(napi_env env, napi_callback_info info, uint32_t (*checksum
     }
     uint8_t *buffer = to_hash.buffer;
     size_t length = to_hash.len;
-    uint32_t previous = 0;
+    uint32_t previous_crc = 0;
 
     if (!aws_napi_is_null_or_undefined(env, node_args[1])) {
-        if (napi_get_value_uint32(env, node_args[1], &previous)) {
-            napi_throw_type_error(env, NULL, "previous argument must be undefined or a positive number");
+        if (napi_get_value_uint32(env, node_args[1], &previous_crc)) {
+            napi_throw_type_error(env, NULL, "previous value argument must be undefined or a positive number");
```

Comment:
I think this error is meant to point the variable name in nodejs function [here](https://github.com/awslabs/aws-crt-nodejs/pull/583/files#diff-21a8783999cf20978be916d17129b69a5a45f60d7d20102cf25b7681501ad17bR51)

Maybe rename the nodejs variable name to `previous_crc`, and use `previous_crc` here as well?

Reply:
im on a fence about it. previous_crc is a local variable that does not have much meaning to caller in js land. from that perspective i think just referring to it as previous value makes more sense

Reply:
mostly because all the other place we have the error throw out is referring to the variable name used in the nodejs land. I think it's better to be consistent